codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -73,6 +78,18 @@ private HAProxyMessage(
             HAProxyProtocolVersion protocolVersion, HAProxyCommand command, HAProxyProxiedProtocol proxiedProtocol,
             String sourceAddress, String destinationAddress, int sourcePort, int destinationPort) {
 
+        this(protocolVersion, command, proxiedProtocol,
+             sourceAddress, destinationAddress, sourcePort, destinationPort, Collections.EMPTY_LIST);
+    }
+
+    /**
+     * Creates a new instance
+     */
+    private HAProxyMessage(
```

Comment:
What is the preferred Netty way for avoid this netkins issue? Should a builder pattern be introduced? for the HAProxyMessage?